scribble-doc/scribblings/scribble/core.scrbl

```diff
@@ -1421,7 +1444,32 @@ number for each subsection. If @racket[numberer-step] produces a plain
 string for the rendered number, then it is not added as a prefix to
 subsection numbers. See also @racket[collected-info].
 
-@history[#:added "6.4"]}
+@history[#:added "1.1"]}
+
+
+@defstruct[link-render-style ([mode (or/c 'default number)])]{
+
+Used as a @tech{style property} for a @racket[part] or a specific
+@racket[link-element] to control the way that the link is rendered.
+
+@margin-note{The @racket[figure-ref] and @racket[Figure-ref] functions
+             from @racketmodname[scriblib/figure] also adapt to
+             a @racket[link-render-style] mode.}
+
+@history[#:added "1.25"]}
+
+
+@defparam[current-link-render-style style link-render-style?]{
+
+A parameter that determines the default rendering style for a section
```

Comment:
Replace "section link" with `link-element` ? (Maybe there should be a `@tech` for this)